ga4gh/frontend.py

```diff
@@ -19,9 +19,15 @@
 import ga4gh.protocol as protocol
 import ga4gh.exceptions as exceptions
 
+import oic
```

Comment:
This group of commits should be included in the third group (i.e., with flask etc). We're organising imports as futures, standard lib, third party, local imports.